tests/admin_views/models.py

```diff
@@ -623,6 +623,27 @@ def __str__(self):
         return self.name
 
 
+class Student(models.Model):
+    name = models.CharField(max_length=100)
+    age = models.PositiveIntegerField()
+
+
+class Course(models.Model):
+    DIFFICULTY_CHOICES = [
+        ("beginner", "Beginner Class"),
+        ("intermediate", "Intermediate Class"),
+        ("advanced", "Advanced Class"),
+    ]
+
+    title = models.CharField(max_length=100)
+    materials = models.FileField(upload_to="test_upload")
+    difficulty = models.CharField(
+        max_length=20, choices=DIFFICULTY_CHOICES, null=True, blank=True
+    )
+    students = models.ManyToManyField(Student, blank=True)
```

Comment:
To reduce the number of models, this could be:
```suggestion
    categories = models.ManyToManyField(Category, blank=True)
```
And we remove the student model

If we can avoid adding a new model entirely, that would also be good